### PR TITLE
Improve headers toolbar

### PR DESCRIPTION
**Pull request by SergioEstevao**
Related to #324 

This PR adds support for selection of the header type to apply when using the format toolbar.
It uses a custom input view that allows to choose what header the user wants to apply.

How to test:
 - Start the demo app
 - Start the editor with content
 - Tap on one of the paragraphs with header
 - Tap on the Header button on the toolbar
 - Check if the selection made on the input view is correct
 - Change the style and see if the style is correct.

### CODE REVIEW

File: Example/Example/EditorDemoController.swift [hunk 1]
```diff
@@ -268,6 +270,13 @@ class EditorDemoController: UIViewController {
         } else {
             identifiers = richTextView.formatIdentifiersForTypingAttributes()
         }
+        // Filter multiple header identifier to single header identifier
+        identifiers = identifiers.map({ (identifier) -> FormattingIdentifier in
+            switch identifier {
+            case .header1, .header2, .header3, .header4, .header5, .header6: return .header
+            default: return identifier
```
Comment (jleandroperez) [inline Example/Example/EditorDemoController.swift]: can we move the `return` (in both cases) to a newline, to aid readability?

File: Example/Example/EditorDemoController.swift [hunk 2]
```diff
@@ -406,9 +416,53 @@ extension EditorDemoController : Aztec.FormatBarDelegate {
...
+            .header3 : .h3,
+            .header4 : .h4,
+            .header5 : .h5,
+            .header6 : .h6,
+        ]
+        for (key,value) in mapping {
+            if identifiers.contains(key) {
+                return value
+            }
+        }
```
Comment (jleandroperez) [inline Example/Example/EditorDemoController.swift]: (No need to change anything!!!) This one can be also synthesized as:

```
        for (key,value) in mapping where identifiers.contains(key) {
            return value
        }
```

File: Example/Example/OptionsTableView.swift [hunk 3]
```diff
@@ -0,0 +1,61 @@
+
+
+import UIKit
```
Comment (jleandroperez) [inline Example/Example/OptionsTableView.swift]: I (think) in the other sources the `import` goes in the first line?

File: Example/Example/OptionsTableView.swift [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+        super.init(frame: frame, style: .plain)
+        self.delegate = self
+        self.dataSource = self
+    }
+
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+
+
```
Comment (jleandroperez) [inline Example/Example/OptionsTableView.swift]: Extra new lines!

File: Example/Example/EditorDemoController.swift (hunk 1 above)
Comment (SergioEstevao) [inline Example/Example/EditorDemoController.swift]: Will do.

**Comment by SergioEstevao** [comment]
@jleandroperez ready for another look.

**Comment by jleandroperez** [comment]
:shipit:

**Pull request merged**